src/components/form/HdRadioIndicator.vue

```diff
@@ -0,0 +1,118 @@
+<script>
+export default {
+  name: 'HdRadioIndicator',
+  props: {
+    checked: {
+      type: Boolean,
+      required: true,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    invalid: {
+      type: Boolean,
+      default: false,
+    },
+  },
+};
+</script>
```

Comment:
Is there any specific reason to start with `script` instead of `template`? I like it btw =D

Reply:
Oh my, thanks for spotting that, I didn't notice that, I have a boilerplate in my code editor, but it was created with `scripts` in the first place, I already changed that, since we are using `templates` first 

e69e869